k8s.gcr.io/images/k8s-staging-coredns/images.yaml

```diff
@@ -14,3 +14,4 @@
     "sha256:b82e294de6be763f73ae71266c8f5466e7e03c69f3a1de96efd570284d35bb18": ["v1.9.4"]
     "sha256:017727efcfeb7d053af68e51436ce8e65edbc6ca573720afb4f79c8594036955": ["v1.10.0"]
     "sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e": ["v1.10.1"]
+    "sha256:cc3ebb05fbdba439d2d69813f162aa204b027098c8244fb3156e6e7c0f31c548": ["v1.11.0"]
```

Comment:
You have to update the image in `registry.k8s.io/images/k8s-staging-coredns`, not in `k8s.gcr.io/images/k8s-staging-coredns/images.yaml`.

Reply:
yes, i corrected it. -:)